src/drawer.multicanvas.js

```diff
@@ -231,15 +231,15 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
 
         // Always draw a median line
         this.fillRect(0, halfH + offsetY - this.halfPixel, this.width, this.halfPixel);
-    },
+    }),
 
-    drawLine: function (peaks, absmax, halfH, offsetY, start, end) {
+    drawLine: utils.frame(function(peaks, absmax, halfH, offsetY, start, end) {
```

Comment:
`utils` is undefined. It has to be `WaveSurfer.util`. Please also restore the spaces after the function keyword.